enterprise-ws-spec/src/main/asciidoc/ServerProgrammingModel.adoc

```diff
@@ -229,12 +222,12 @@ is used to guarantee uniqueness
 |===
 
 For Stateless or Singleton Session EJBs using this annotation, the name
-attribute of the javax.ejb.Stateless or javax.ejb.Singleton annotation
+attribute of the jakarta.ejb.Stateless or jakarta.ejb.Singleton annotation
 on the Service Implementation Bean class must be used as the <ejb-link>
 element in the deployment descriptor to map the Port component to the
-actual EJB. If name attribute in javax.ejb.Stateless or
-javax.ejb.Singleton annotation is not specified, then the default value
-is used as defined in the section 4.4.1 of EJB 3.1.
+actual EJB. If name attribute in jakarta.ejb.Stateless or
+jakarta.ejb.Singleton annotation is not specified, then the default value
+is used as defined in the Jakarta Enterprise Beans 4.0 specification.
```

Comment:
I would not remove reference to particular section of the spec. Probably also omit spec version number.

Reply:
Will remove spec version 4.0 and add the section numbers